Sparky-core/src/sp/scripting/script.h

```diff
@@ -30,7 +30,16 @@ namespace sp { namespace scripting {
 
 		template <typename... Args>
 		void CallFunction(const char* func, Args&&... args) {
-			sp::scripting::FunctionCaller::Dispatch(m_State, func, args...);
+			try
+			{
+				luabind::call_function<void>(m_State, func, args...);
+			}
+			catch (luabind::error& e)
+			{
+				SP_ERROR("Lua >> ", lua_tostring(m_State, -1));
+			}
+			
```

Comment:
Personally I wouldn't recommend surpressing the exception. Either notify the caller about an error with a bool value or throw an own exception.